.github/workflows/evm-benchmark.yml

```diff
@@ -0,0 +1,52 @@
+name: EVM benchmark
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - core/vm/**
+  workflow_dispatch:
```

Comment:
Not sure we can run 2 modes in same file or not. If not,  We need to separate 2 files, one for
```
on:
  pull_request:
    branches:
      - master
    paths:
      - core/vm/**
 ```

one for workflow_dispatch
```
on:
  workflow_dispatch:
```

then remove if logic.